<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipeDialog</class>
 <widget class="QDialog" name="AddRecipeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1488</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="recipeNameLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="prepTimeEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ingredientsTable">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>291</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addIngredientButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Ingredient</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="stepsTextEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
